helm/frost-server/templates/http-deployment.yaml

```diff
@@ -108,9 +108,18 @@ spec:
             - name: persistence_db_jndi_datasource
               value: ""
             - name: persistence_db_driver
+              value: "org.postgresql.Driver"
+            {{ if .Values.frost.db.enableIntegratedDb }}
+              value: "{{ .Values.frost.db.driver }}"
+              value: "org.postgresql.Driver"
+            {{ if .Values.frost.db.enableIntegratedDb }}
               value: "{{ .Values.frost.db.driver }}"
             - name: persistence_db_url
               value: {{ printf "jdbc:postgresql://%s:5432/%s" (include "frost-server.fullName" (merge (dict "tier" "db") .)) .Values.frost.db.database | quote }}
+            {{ else }}
+            - name: persistence_db_url
+              value: "{{ .Values.frost.db.dbExternalConnectionString }}"
+            {{ end }}
```

Comment:
It seems these bits went sideways while merging. It's probably best to always have the `persistence_db_driver` option (in case the external connection is a non-postgres) and have the `persistence_db_url` in the if/else.